turbinia/workers/bulk_extractor.py

```diff
@@ -72,6 +72,24 @@ def run(self, evidence, result):
       return result
     return result
 
+  def check_xml_attrib(self, xml_key):
+    """Checks if a key exists within the xml report
+
+    Args: 
+      xml_key(str): the xml key to check for.
+    
+    Returns:
+      xml_hit(str): the xml value else return N/A.
+    """
+    xml_hit = "N/A"
+    xml_search = self.xml.find(xml_key)
+
+    # If exists, return the text value.
+    if xml_search is not None:
+      xml_hit = xml_search.text
+      return xml_hit
```

Comment:
Since you defined `xml_hit` above, you should't need to add an extra return statement here.

Reply:
Removed the extra return statement